Task 1/js/index.js

```diff
@@ -1,57 +1,63 @@
 import Polyfill from "./modules/polyfill.js";
-import findUnique from "./modules/findUnique.js"
+import findUnique from "./modules/findUnique.js";
 
-const notes = [
-{
+const NOTES = [
+  {
     id: 1,
     title: "Recipe",
     description: "Ingredients include 2 eggs...",
     pagesCount: 2,
     isMarked: false,
-    access: []
-},
-{
+    access: [],
+  },
+  {
     id: 2,
     title: "Workouts",
     description: "3 sets of squats...",
     pagesCount: 1,
     isMarked: true,
-    access: []
-},
-{
+    access: [],
+  },
+  {
     id: 3,
     title: "Passwords",
     description: "VISA ...",
     pagesCount: 6,
     isMarked: true,
-    access: []
-},
-{
+    access: [],
+  },
+  {
     id: 4,
     title: "To Do 2021",
     description: "1. Learn JS...",
     pagesCount: 3,
     isMarked: false,
-    access: []
-}];
+    access: [],
+  },
+];
 
-const polyfill = new Polyfill(notes);
+const polyfill = new Polyfill(NOTES);
 
-const filterArray = polyfill.customFilter((item) => item.isMarked);
+const filterArray = polyfill.customFilter((newArray) => newArray.isMarked);
```

Comment:
It's not an array as a parameter in arrow function, it's an object!
Rename it please

Reply:
`note`